suites/pacific/rados/tier-3_rados_ssd-tests.yaml

```diff
@@ -162,14 +198,14 @@ tests:
               command: add
               service: osd
               pos_args:
-                - node4
-                - "/dev/sdc"
+                - node14
+                - "/dev/sdm"
```

Comment:
Let's add a small check to see if the device is listed as available in `ceph orch device ls --refresh`. That way we can make sure if the drive is available for OSD deployment. 

In case if that is not available, pick a free disk in same host for deployment.

Reply:
Added the check before configuring the OSD.